pkg/utils/kube/kube_test.go

```diff
@@ -167,6 +166,170 @@ spec:
 	assert.Nil(t, GetDeploymentReplicas(&noDeployment))
 }
 
+func TestGetResourceImages(t *testing.T) {
+	type testCase struct {
+		manifest    []byte
+		expected    []string
+		description string
+	}
+
+	testCases := []testCase{
+		{
```

Comment:
```suggestion
	type testCase struct {
		manifest    []byte
		expected    []string
		description string
	}{
		{
```

We can probably add the no images tests here right? By just expecting an empty/nil array?